sources/packages/web/src/components/common/StudentApplicationCommonActions.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-row>
+    <v-col md="6">
+      <v-btn
+        color="primary"
+        v-show="!isFirstPage"
+        outlined
+        @click="$emit('wizardGoPrevious')"
+        >Previous section</v-btn
+      >
+    </v-col>
+    <v-col md="6" class="ml-auto text-right">
```

Comment:
@ann-aot when we spoke during sync up our plan to keep these actions inside the application common component right? Is there any specific reason to keep actions as separate component?

Reply:
Yeah got that it is the child of common component itself. Just trying to understand, Is there any specific reason to keep actions as separate component?

Reply:
since in the student application all action buttons are inside vue, i wanted to keep it as a separate action component specific for the student application, that was my reason,  but updated the code